README.md

```diff
@@ -66,16 +66,70 @@ row_model.name # => "SOME PROJECT NAME"
 
 `Import::File` also provides the `RowModel` with the previous `RowModel` instance:
 
+```ruby
+row_model.previous # => <ProjectImportRowModel instance>
+row_model.previous.previous # => nil, save memory by avoiding a linked list
 ```
+
+## Presenter
+For complex rows, you can wrap your `RowModel` with a presenter:
+
+```ruby
+class ProjectImportRowModel < ProjectRowModel
+  include CsvRowModel::Import
+
+  # same as above
+
+  presenter do
```

Comment:
👍🏻 I think is easier to read the import model... Maybe some clases can become really big though